hack/generator/pkg/codegen/code_generator.go

```diff
@@ -76,9 +63,17 @@ func (generator *CodeGenerator) Generate(ctx context.Context) error {
 		return errors.Wrapf(err, "failed to walk JSON schema")
 	}
 
-	defs, err = generator.FilterDefinitions(defs)
-	if err != nil {
-		return errors.Wrapf(err, "failed to filter generated definitions")
+	pipeline := []PipelineStage{
+		applyExportFilters(generator.configuration),
+		deleteGeneratedCode(ctx, generator.configuration.OutputPath),
```

Comment:
Perhaps, `ctx` would be better served to be on the action rather than the constructor of the structure.

Reply:
Agreed, will change.

Reply:
Done.